data/source/countries.csv

```diff
@@ -220,7 +220,7 @@ SUI,41,CHE,sz,Yes,756,237,SZ,SW,CH,SUI,SUI,CH,la Confederación Suiza,True,,Suis
 TJK,992,TJK,ta,Yes,762,239,TI,TA,TJ,TJK,TJK,TJ,la República de Tayikistán,True,,Tadjikistan,Tadjikistan (le),Somoni,Developing,Республика Таджикистан,Tajikistan,TJS,,Tayikistán,972,塔吉克斯坦共和国,la République du Tadjikistan,Таджикистан,762,143,142,طاجيكستان,2,جمهورية طاجيكستان,塔吉克斯坦,x,,Tayikistán,the Republic of Tajikistan,塔吉克斯坦,Tajikistan,TAJIKISTAN,,Asia,طاجيكستان,Central Asia,Таджикистан,World,Dushanbe,AS,.tj,"tg,ru",1220409,Tajikistan,2D
 THA,66,THA,th,Yes,764,240,TH,TH,TH,THA,THA,T,el Reino de Tailandia,True,,Thaïlande,Thaïlande (la),Baht,Developing,Королевство Таиланд,Thailand,THB,,Tailandia,764,泰王国,le Royaume de Thaïlande,Таиланд,764,35,142,تايلند,2,مملكة تايلند,泰国,,,Tailandia,the Kingdom of Thailand,泰国,Thailand,THAILAND,,Asia,تايلند,South-eastern Asia,Таиланд,World,Bangkok,AS,.th,"th,en",1605651,Thailand,W1
-MKD,389,MKD,xn,Yes,807,241,MK,MJ,MK,MKD,MKD,MK,ex República Yugoslava de Macedonia (la),True,,ex-République yougoslave de Macédoine,ex-République yougoslave de Macédoine (l') [fém.],Denar,Developed,бывшая югославская Республика Македония,the former Yugoslav Republic of Macedonia,MKD,,ex República Yugoslava de Macedonia,807,前南斯拉夫的马其顿共和国,l'ex-République yougoslave de Macédoine [fém.],бывшая югославская Республика Македония,807,39,150,جمهورية مقدونيا اليوغوسلافية سابقاً,2,جمهورية مقدونيا اليوغوسلافية سابقاً,前南斯拉夫的马其顿共和国,x,,ex República Yugoslava de Macedonia,the former Yugoslav Republic of Macedonia,前南斯拉夫的马其顿共和国,The former Yugoslav Republic of Macedonia,MACEDONIA (THE FORMER YUGOSLAV REPUBLIC OF),,Europe,جمهورية مقدونيا اليوغوسلافية سابقاً,Southern Europe,бывшая югославская Республика Македония,World,Skopje,EU,.mk,"mk,sq,tr,rmm,sr",718075,North Macedonia,
+MKD,389,MKD,xn,Yes,807,241,MK,MJ,MK,MKD,MKD,MK,ex República Yugoslava de Macedonia (la),True,,ex-République yougoslave de Macédoine,ex-République yougoslave de Macédoine (l') [fém.],Denar,Developed,бывшая югославская Республика Македония,North Macedonia,MKD,,ex República Yugoslava de Macedonia,807,前南斯拉夫的马其顿共和国,l'ex-République yougoslave de Macédoine [fém.],бывшая югославская Республика Македония,807,39,150,جمهورية مقدونيا اليوغوسلافية سابقاً,2,جمهورية مقدونيا اليوغوسلافية سابقاً,前南斯拉夫的马其顿共和国,x,,ex República Yugoslava de Macedonia,North Macedonia,前南斯拉夫的马其顿共和国,North Macedonia,MACEDONIA (THE FORMER YUGOSLAV REPUBLIC OF),,Europe,جمهورية مقدونيا اليوغوسلافية سابقاً,Southern Europe,бывшая югославская Республика Македония,World,Skopje,EU,.mk,"mk,sq,tr,rmm,sr",718075,North Macedonia,
```

Comment:
Should these other language versions be re-translated? The ones I can make out all seem to be variants on the former name of the country.